docs/snippets/common/my-addon-storybook-registration.js.mdx

```diff
@@ -7,7 +7,7 @@ module.exports = {
     "@storybook/addon-links",
     "@storybook/addon-essentials",
     "@storybook/preset-create-react-app",
-    "../src/preset.js" //👈 Our addon registered here
+    "../dist/preset.js" //👈 Our addon registered here
```

Comment:
here dist is the minified/concatenated version  ( used on production sites)

src  is the raw code(used to read/edit the code)

i think this is the change made here because  its addon registered written .

Reply:
@Astrodevil thank you for the comment on the pull request, but documentation flow wise at this stage the reader is still in "development mode" (pardon the bad pun), the addon is still early on. Applying the change here would involve having to build it first. And as the Storybook's  addon capabilities and possibilities are far and wide, we cannot know for sure if the case applied to the author is valid for other cases and including this change can lead to some misleadings. Also in all fairness it's best to reference the changed introduced with this pull request in a [root level preset](https://storybook.js.org/docs/react/addons/writing-addons#root-level-presetjs) , that will make the life easier for the addon consumers.

Stay safe

Reply:
Hi all,

This code change is in reference to the documentation for [writing addons](https://storybook.js.org/docs/react/addons/writing-addons#register-the-addon). I did a search through the codebase to find reference to this snippet so let me know if the change should be made somewhere else.

This snippet is referencing `/my-addon/.storybook/main.js` which is entirely local to the addon developer and not part of the published package (let me know if there is a case where it is part of the production package).

The [documentation example](https://storybook.js.org/docs/react/addons/writing-addons#build-system) uses ESM (`import/export`) in `src` but `./storybook/main.js` file cannot reference it and thus must use the transpiled CJS export (note how `main.js` is using `module.exports` and not `export default`).